lib/cide.rb

```diff
@@ -208,7 +188,7 @@ def clean
         return
       end
 
-      x = run("docker inspect #{cide_image_ids.join(' ')}", capture: true)
+      x = docker('inspect', *cide_image_ids, capture: true)
```

Comment:
fair. since these changes don't introduce the x variable i'm going to let that for a future change